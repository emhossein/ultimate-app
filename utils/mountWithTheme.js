import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from '@emotion/react';
import theme from '../config/theme';

const mountWithTheme = ({ children }) => {
  return renderer.create(
    <ThemeProvider theme={theme}>{children}</ThemeProvider>
  );
};

export default mountWithTheme;
