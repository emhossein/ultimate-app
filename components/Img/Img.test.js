/**
 * @jest-environment jsdom
 */

import React from 'react';
import { createSerializer } from '@emotion/jest';
import mountWithTheme from '../../utils/mountWithTheme';
import { Img } from '..';

expect.addSnapshotSerializer(createSerializer());

describe('HyperLink', () => {
  test('Renders Correctly', () => {
    const component = mountWithTheme(<Img src="image" alt="image" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
