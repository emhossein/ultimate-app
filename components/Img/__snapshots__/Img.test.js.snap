// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HyperLink Renders Correctly 1`] = `null`;
